src/modules/Lan.rb

```diff
@@ -651,12 +651,12 @@ def ProposeVirtualized
     #
     # @return [Array] of packages needed when writing the config
     def Packages
-      pkgs = []
-      backend = yast_config&.backend
-
-      return pkgs unless backend
+      backend = Mode.autoinst ? autoinst.selected_backend : yast_config&.backend
+      return [] unless backend
 
-      backend.packages.each { |p| pkgs << p if !Package.Installed(p) }
+      pkgs = []
+      target = Mode.autoinst ? :autoinst : :system
+      backend.packages.each { |p| pkgs << p if !Package.Installed(p, target: target) }
```

Comment:
What about a more idiomatic way? And we could use a better name for the variable.

```ruby
# ok, the name sucks, but it is more descriptive
non_installed_pkgs = backend.packages.reject { |p| Package.Installed(p, target: target) }
```